build.boot

```diff
@@ -67,11 +67,11 @@
 
     ;; ------- Deps for project repl ------------------
     ;; The following dependencies are from: https://github.com/adzerk-oss/boot-cljs-repl
-    [adzerk/boot-cljs-repl   "0.3.3"] ;; latest release
-    [com.cemerick/piggieback "0.2.2"  :scope "test"]
-    [weasel                  "0.7.0"  :scope "test"]
-    [org.clojure/tools.nrepl "0.2.13" :scope "test"]
-    [cljsjs/babel-polyfill "6.20.0-2" :scope "test"]
+    [adzerk/boot-cljs-repl   "0.4.0"]
+    [cider/piggieback        "0.3.9"]
+    [weasel                  "0.7.0"]
+    [nrepl                   "0.4.5"]
+    ;; [cljsjs/babel-polyfill "6.20.0-2" :scope "test"]
```

Comment:
@bago2k4 is this polyfill still strictly necessary? It doesn't appear to be documented in the [boot-cljs-repl](https://github.com/adzerk-oss/boot-cljs-repl) readme anymore.